recipes/f77_zmq/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "f77_zmq" %}
+{% set version = "4.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zeromq/f77_zmq/releases/download/v4.3.2_autotools/f77-zmq-4.3.2.tar.gz
+  sha256: e027bf1017ca6b47ce4a3481ae5b255db8ea86294bc32f1207bd90dbdece7671
+
+build:
+  skip: true  # [win]
+  number: 0
```

Comment:
Please add ignore_run_exports for the gfortran library. [Since it is not needed at runtime.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=486601&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1309)